usaspending_api/download/v2/request_validations.py

```diff
@@ -274,3 +277,12 @@ def _validate_required_parameters(request_data, required_parameters):
     for required_param in required_parameters:
         if required_param not in request_data:
             raise InvalidParameterException("Missing one or more required body parameters: {}".format(required_param))
+
+
+def _validate_file_formats(json_request: dict) -> str:
```

Comment:
This function is a violation of Single Responsibility Principle. Is there any reason we can't move the exception logic into the existing `_validate_filters` function, and return this code to just being the default logic?

Reply:
This follows the current workflow in this file. Ironically it is similar to `_validate_filters()`.

I'm not agreeing with the pattern, but it seems out of scope to refactor all of the validation logic for this new validation.

Reply:
As far as I can tell this would only be a few lines being moved from one function to another, and would prevent things from getting further obfuscated. I don't see any other lines using a "validate" function to generate a value.

Reply:
`_validate_account_level()`?

Reply:
...nevermind